### PR TITLE
Links and speaker page

### PR DESCRIPTION
**Pull request by dotemacs**
Added a link to the Remacs talk, sorry I missed this!

Added a little link, to encourage people to speak.

Added a page for remote speakers, with the questions that come up often.

### CODE REVIEW

**Review by Wilfred — APPROVED** [review]
Oh, I didn't even think about my transcript. Woops!

Do you want to link to the remote speaker page from the home page?

File: for-remote-speakers.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	</ul>
+	<p>
+	  It would be good to have all of the above ready so that if
+	  one approach fails, we can fail over to the next.
+	</p>
+
+	<h2>How long can I speak for?</h2>
+	<p>
+          Tell us how long do you need. Usually we have the venue for
+          90-ish minutes.
```
Comment (Wilfred) [inline for-remote-speakers.html]: This implies to me that speakers usually talk for ~90 mins. Perhaps say 'with typically 2-3 speakers'.

Comment (dotemacs) [reply for-remote-speakers.html]: Will do, thanks.

**Comment by dotemacs** [comment]
Added a bit more details about the talk duration, as per your suggestion.

As for the page about the speakers, not sure if it needs to be linked from the main page. Maybe that can be just shared with the speakers... what do you think?

I'll merge what we've got so far, we can always modify it later.

**Pull request merged**